src/libraries/src.proj

```diff
@@ -5,20 +5,24 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <_allSrc Include="$(MSBuildThisFileDirectory)*\src\*.csproj"
-             Exclude="@(ProjectExclusions)" />
-    <!-- Don't build ProjectReferences of the compat pack in a traversal build to avoid unnecessary incremental builds. -->
-    <_allSrc Update="$(MSBuildThisFileDirectory)Microsoft.Windows.Compatibility\src\Microsoft.Windows.Compatibility.csproj"
-             AdditionalProperties="BuildProjectReferences=false" />
+    <_allSrc Include="$(MSBuildThisFileDirectory)*\src\*.*proj"
+             Exclude="@(ProjectExclusions);
+                      $(MSBuildThisFileDirectory)*\src\**\*.shproj" />
     <!-- Only build the compat pack in the allconfigurations leg. -->
     <_allSrc Remove="$(MSBuildThisFileDirectory)Microsoft.Windows.Compatibility\src\Microsoft.Windows.Compatibility.csproj"
              Condition="'$(BuildAllConfigurations)' != 'true'" />
+
     <NonNetCoreAppProject Include="@(_allSrc)"
-                          Exclude="@(NetCoreAppLibrary->'%(Identity)\src\%(Identity).csproj')" />
+                          Exclude="@(NetCoreAppLibrary->'%(Identity)\src\%(Identity).csproj');
+                                   $(MSBuildThisFileDirectory)Microsoft.VisualBasic.Core\src\Microsoft.VisualBasic.Core.vbproj;
```

Comment:
why? these two are part of the shared framework no?

Reply:
Right, that's why we are excluding them from the NonNetCoreAppProject. The Glob filter doesn't allow a .*proj in this case. That was already there before.

Reply:
Duh, I read wrong. I read Include=... thanks for solving my distracted question 😆